docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM docker.io/fedora:27
 
 RUN dnf -y install \
-        cmake \
```

Comment:
The reason why circle.ci fails to build this successfully, is that you haven't pushed the changes to the docker container?

Reply:
No idea, I thought changing the Dockerfile would have been enough.

Reply:
Let me take care of that :)